src/badguy/badguy.cpp

```diff
@@ -181,6 +183,15 @@ BadGuy::update(float dt_sec)
 {
   if (m_frozen && !is_grabbed())
   {
+    Rectf playerbox = get_bbox().grown(-2.f);
+    playerbox.set_bottom(get_bbox().get_bottom() + 7.f);
+    for (auto& player : Sector::get().get_objects_by_type<Player>())
+    {
+      if (playerbox.overlaps(player.get_bbox()) && m_physic.get_velocity_y() > 0.f && is_portable()) {
+        m_physic.set_velocity_y(-250.f);
+      }
+    }
+
```

Comment:
What does this do?

Reply:
Makes the frozen enemies bounce off Tux's head like rocks.